### DISCUSSION TITLE
SMS Sender.

### DISCUSSION THREAD

**Discussion by corsacca** [DiscipleTools/disciple-tools-theme#2866] [Integrations]
I'm opening this issue to collect the requirements for sms integration.
The main purpose of the integration is to create sms campaign that that send texts to contacts that match a filter.
ie send 3 text to on pause contacts that are from source "web"

**Comment by corsacca** [comment]
Here are the current mockups.
![image](https://user-images.githubusercontent.com/24901539/39256765-a7258f2e-48a7-11e8-8ffd-1345e8d23af2.png)
![image](https://user-images.githubusercontent.com/24901539/39256786-b021fb3a-48a7-11e8-9bd2-39e9222063e8.png)

**Comment by jlebouch76** [comment]
We could filter a particular set of contacts (from the Contact Menu), save it by a name and later on, I the SMS campaign create a particular SMS campaign for them.
In your mockup, I don't understand the Pre-requisite.

It will also be great to find the SMS campaign name in the Contact's profile. It's important for the user to see what kind of SMS campaign was sent and when.

Another thing would be to have SMS campaign history, and a place for Scheduling the SMS campaigns.
